plugins/destination/postgresql/client/list_tables.go

```diff
@@ -53,9 +53,8 @@ ORDER BY
 	table_name ASC, ordinal_position ASC;
 `
 
-func (c *Client) listTables(ctx context.Context, include, exclude []string) (schema.Tables, error) {
-	var tables schema.Tables
-	whereClause := c.whereClause(include, exclude)
+func (c *Client) getDBTable(ctx context.Context, name string) (*schema.Table, error) {
+	whereClause := "AND pg_class.relname = '" + inClause(name) + "'"
```

Comment:
```suggestion
	whereClause := "AND pg_class.relname IN " + inClause(name)
```
I don't think this complex code is necessary as the migrate table is invoked separately for each table